x/emissions/keeper/inference_synthesis/synth_palette_weight.go

```diff
@@ -64,23 +65,34 @@ func (p *SynthPalette) CalcWeightsGivenWorkers() (RegretInformedWeights, error)
 		}
 	}
 
+	moreThanOneNotNewInferer := p.SingleInfererNotNew == ""
```

Comment:
you use this condition check in multiple places. You should probs have a palette method for it